src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+const body = document.querySelector('body');
+const div = document.createElement('div');
+
+div.dataset.qa = 'notification';
+
+let firstClick = false;
+let secondClick = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  const delay = setTimeout(() => {
+    div.classList.add('warning');
+    div.innerText = 'First promise was rejected';
+    body.append(div);
```

Comment:
create a separate function for adding the notification and reuse it everywhere. please create the `div` in that function each time you call it.